app/scripts/lib/message-manager.js

```diff
@@ -217,7 +218,16 @@ export default class MessageManager extends EventEmitter {
    * @param {number} msgId - The id of the Message to reject.
    *
    */
-  rejectMsg(msgId) {
+  rejectMsg(msgId, reason = undefined) {
+    if (reason) {
+      this.metricsEvent({
```

Comment:
We will need to replicate this pattern in the other signature message managers as well (`personalMessageManager` and `typedMessageManager`). All three of these message managers are used by the signature request confirmations.